pkgs/by-name/tp/tpm-fido/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  pinentry-tty,
+}:
+
+buildGoModule {
+  pname = "tpm-fido";
+  version = "0-unstable-2024-10-30";
+
+  src = fetchFromGitHub {
+    owner = "psanford";
+    repo = "tpm-fido";
+    rev = "5f8828b82b58f9badeed65718fca72bc31358c5c";
+    hash = "sha256-Yfr5B4AfcBscD31QOsukamKtEDWC9Cx2ee4L6HM2554=";
+  };
+
+  vendorHash = "sha256-qm/iDc9tnphQ4qooufpzzX7s4dbnUbR9J5L770qXw8Y=";
+
+  buildInputs = [
+    pinentry-tty
+  ];
+
+  ldFlags = [
+    "-s "
+    "-w"
+  ];
+
+  meta = {
```

```suggestion
  ];

  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  versionCheckProgramArg = [ "--version" ];
  doInstallCheck = true;

  meta = {
```
Can you maybe try to add this please?

---

Ok thanks !